### DISCUSSION TITLE
How can we create such interface in laravel nova?

### DISCUSSION THREAD

**Discussion by beshoo** [laravel/nova-issues#5752] [General]
![image](https://github.com/laravel/nova-issues/assets/4635115/27fef063-2c9e-4e92-a523-9ca53f172c9c)

Note the photo on the right side,
Note the fields how placed horizontally....

Nova has to support such look without need looking for add on

**Comment by davidhemphill** [comment]
Nova doesn't currently support an arbitrary arrangement of field panels and doesn't have a field similar to the one on the right of the screenshot.

Additionally, it's been pretty well-tested in user research for years that horizontal form fields are confusing for users, so we've avoided supporting them. However, you can create a custom field that displays fields in whatever format makes sense for your application.
